### ISSUE TITLE
Centeur emacs org preview show internal error

### ISSUE THREAD

**Issue by ZhengshuaiPENG** [seagle0128/grip-mode#13]
Hi seagle0218,

I am using centaur emacs now and I want to preview my org file.
But when I use `M-x grip-mode`, the web page shows me 500
```
Internal Server Error
The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.
```
And I saw there was a md file generated.

I already installed python3 and grip in my system (OSX), is there any way to debug it?

And here are outputs in the message buffer
```
Saving file /Users/zhengshuai.peng/Workspace/notes/ProductResearch/Hologres/hologres_summary.md...
Wrote /Users/zhengshuai.peng/Workspace/notes/ProductResearch/Hologres/hologres_summary.md
Preview ‘/Users/zhengshuai.peng/Workspace/notes/ProductResearch/Hologres/hologres_summary.org’ on http://localhost:61059
mwheel-scroll: Beginning of buffer
Quit
Process ‘grip-61059’ killed
```

**Comment by ZhengshuaiPENG** [comment]
In addition, I saw the command in emacs like 
![image](https://user-images.githubusercontent.com/11333496/89986924-d0482d00-dcaf-11ea-9f13-58c0798a39a3.png)

grip version: 4.5.2
It seems that `- Grip` is not supported in grip command

**Comment by seagle0128** [comment]
The command is incorrect. user and password are not set. I have no idea why they are nil.

```elisp
;; A GitHub username for API authentication
(setq grip-github-user "")

;; A GitHub password or auth token for API auth
(setq grip-github-password "")
```

**Comment by ZhengshuaiPENG** [comment]
Thanks for your reply
I added these configurations in `custom.el` in my centaur config
![image](https://user-images.githubusercontent.com/11333496/90088860-9714c900-dd52-11ea-8273-16b0efdae23f.png)
I also tried to upgrade all centaur configuration and packages to lastest yesterday. The only changed file is `custom.el` is there any idea?

**Comment by seagle0128** [comment]
1. `M-x list-process`
1. Open grip process buffer like `*grip-xxxx*`.
1. Check the logs.

**Comment by ZhengshuaiPENG** [comment]
Hi seagle0128,

I checked the process log
![image](https://user-images.githubusercontent.com/11333496/90207744-a6ab1500-de19-11ea-91bd-1168fb89360f.png)
Seems that it's an authorization issue.

So the problem is my configuration is not correct? it doesn't take effect when set them in `custom.el`

**Comment by ZhengshuaiPENG** [comment]
I also tried use `(setq-default grip-github-user "xxx")` and `(setq-default grip-github-password "xxx")`. No luck with this way.

Tried another way, use `customize-group`, I saw the `grip-github-user` is `nil`, tried to edit it, but I can't apply and save

![image](https://user-images.githubusercontent.com/11333496/90212618-6aca7c80-de26-11ea-991d-f8270ed0d70b.png)

**Comment by ZhengshuaiPENG** [comment]
**Rootcause Found**
In `.emacs.d/lisp/init-markdown.el`
![image](https://user-images.githubusercontent.com/11333496/90213148-969a3200-de27-11ea-824a-41e9ba11a1e3.png)

If I changed `grip-github-user` and `grip-github-password` here, the preview works.

I don't understand that in this code snipper

```
(setq grip-github-user (car credential)
     grip-github-password (cadr credential)
```

how `(car credential)` to get the user name and the password

**Comment by seagle0128** [comment]
Please update Centaur Emacs.

**Comment by ZhengshuaiPENG** [comment]
Hi, @seagle0128 

Thanks for your update.

I create a `~/.authinfo` and put `login {user_name} password {token}`, the grip mode works.

But I found that if I just set the `(setq grip-github-user "{user_name})` and `(setq grip-github-password "{token}")` in my `custom.el` by following the README, the grip mode still doesn't work.

Is there a way that make grip mode work when I set these in my `custome.el`?
